### DISCUSSION TITLE
ArchR error while initialization ShinyArchRUiO

### DISCUSSION THREAD

**Discussion by susrinivasan** [GreenleafLab/ArchR#1955] [General]
Hi,

I am able to run the ShinyArchRUiO just fine on my local development server. However when I copy the entire ArchR project directory onto my production server and run the initialization, I get the following error.  Can you please help in resolving this?

-----------------------------------------------------
[susrinivasan@sc-bcwgweb ShinyArchRUiO]$ more /var/log/shiny-server/ShinyArchRUiO-shiny-20230518-152337-38651.log
su: ignoring --preserve-environment, it's mutually exclusive with --login
Loading required package: Seurat
Attaching SeuratObject
Loading required package: shinycssloaders
Loading required package: hexbin
Loading required package: magick
Linking to ImageMagick 6.9.12.86
Enabled features: cairo, fontconfig, freetype, ghostscript, lcms, pango, raw, rsvg, webp, x11
Disabled features: fftw, heic
Using 16 threads
Loading required package: gridExtra
Loading required package: grid
Loading required package: patchwork
Loading required package: shinybusy
Loading required package: ArchR

                                                   / |
                                                 /    \
            .                                  /      |.
            \\\                              /        |.
              \\\                          /           `|.
                \\\                      /              |.
                  \                    /                |\
                  \\#####\           /                  ||
                ==###########>      /                   ||
                 \\##==......\    /                     ||
            ______ =       =|__ /__                     ||      \\\
        ,--' ,----`-,__ ___/'  --,-`-===================##========>
       \               '        ##_______ _____ ,--,__,=##,__   ///
        ,    __==    ___,-,__,--'#'  ==='      `-'    | ##,-/
        -,____,---'       \\####\\________________,--\\_##,/
           ___      .______        ______  __    __  .______      
          /   \     |   _  \      /      ||  |  |  | |   _  \     
         /  ^  \    |  |_)  |    |  ,----'|  |__|  | |  |_)  |    
        /  /_\  \   |      /     |  |     |   __   | |      /     
       /  _____  \  |  |\  \\___ |  `----.|  |  |  | |  |\  \\___.
      /__/     \__\ | _| `._____| \______||__|  |__| | _| `._____|
    
ArchR : Version 1.0.2
For more information see our website : www.ArchRProject.com
If you encounter a bug please report : https://github.com/GreenleafLab/ArchR/issues
Loading Required Packages...
        Loading Package : grid v4.3.0
        Loading Package : gridExtra v2.3
        Loading Package : gtools v3.9.4
        Loading Package : gtable v0.3.3
        Loading Package : ggplot2 v3.4.2
        Loading Package : magrittr v2.0.3
        Loading Package : plyr v1.8.8
        Loading Package : stringr v1.5.0
        Loading Package : data.table v1.14.8
        Loading Package : matrixStats v0.63.0
        Loading Package : S4Vectors v0.38.1
        Loading Package : GenomicRanges v1.52.0
        Loading Package : BiocGenerics v0.46.0
        Loading Package : Matrix v1.5.4
        Loading Package : Rcpp v1.0.10
        Loading Package : SummarizedExperiment v1.30.1
        Loading Package : rhdf5 v2.44.0
Loading required package: ggseqlogo
Loading required package: monocle3
Loading required package: SingleCellExperiment

Attaching package: ‘monocle3’

The following objects are masked from ‘package:Biobase’:

    exprs, fData, fData<-, pData, pData<-

Input threads is equal to or greater than ncores minus 1 (15)
Setting cores to ncores minus 2. Set force = TRUE to set above this number!
Setting default number of Parallel threads to 14.
Setting default genome to Mm10.
Successfully loaded ArchRProject!

                                                   / |
                                                 /    \
            .                                  /      |.
            \\\                              /        |.
              \\\                          /           `|.
                \\\                      /              |.
                  \                    /                |\
                  \\#####\           /                  ||
                ==###########>      /                   ||
                 \\##==......\    /                     ||
            ______ =       =|__ /__                     ||      \\\
        ,--' ,----`-,__ ___/'  --,-`-===================##========>
       \               '        ##_______ _____ ,--,__,=##,__   ///
        ,    __==    ___,-,__,--'#'  ==='      `-'    | ##,-/
        -,____,---'       \\####\\________________,--\\_##,/
           ___      .______        ______  __    __  .______      
          /   \     |   _  \      /      ||  |  |  | |   _  \     
         /  ^  \    |  |_)  |    |  ,----'|  |__|  | |  |_)  |    
        /  /_\  \   |      /     |  |     |   __   | |      /     
       /  _____  \  |  |\  \\___ |  `----.|  |  |  | |  |\  \\___.
      /__/     \__\ | _| `._____| \______||__|  |__| | _| `._____|
    
ArchR logging to : ArchRLogs/ArchR-addImputeWeights-289505d13f07f-Date-2023-05-18_Time-15-24-01.970172.log
If there is an issue, please report to github with logFile!
2023-05-18 15:24:02.039044 : Computing Impute Weights Using Magic (Cell 2018), 0 mins elapsed.
ArchR logging to : ArchRLogs/ArchR-addImputeWeights-2895047537686-Date-2023-05-18_Time-15-24-09.261349.log
If there is an issue, please report to github with logFile!
2023-05-18 15:24:09.290386 : Computing Impute Weights Using Magic (Cell 2018), 0 mins elapsed.
ArchR logging to : ArchRLogs/ArchR-addImputeWeights-289502bc0556c-Date-2023-05-18_Time-15-24-17.415933.log
If there is an issue, please report to github with logFile!
2023-05-18 15:24:17.444849 : Computing Impute Weights Using Magic (Cell 2018), 0 mins elapsed.
Getting ImputeWeights
ArchR logging to : ArchRLogs/ArchR-plotEmbedding-289504545676b-Date-2023-05-18_Time-15-24-26.491546.log
If there is an issue, please report to github with logFile!
Imputing Matrix
Warning in mclapply(..., mc.cores = threads, mc.preschedule = preschedule) :
  4 function calls resulted in an error
Error in .safelapply(seq_along(blocks), function(y) { : 
Error Found Iteration 1 : 
        [1] "Error in h(simpleError(msg, call)) : \n  error in evaluating the argument 'x' in selecting a method for function 't': requires nu
meric/complex matrix/vector arguments\n"
        <simpleError in h(simpleError(msg, call)): error in evaluating the argument 'x' in selecting a method for function 't': requires numer
ic/complex matrix/vector arguments>
Error Found Iteration 2 : 
        [1] "Error in h(simpleError(msg, call)) : \n  error in evaluating the argument 'x' in selecting a method for function 't': requires nu
meric/complex matrix/vector arguments\n"
        <simpleError in h(simpleError(msg, call)): error in evaluating the argument 'x' in selecting a method for function 't': requires numer
ic/complex matrix/vector arguments>
Error Found Iteration 3 : 
        [1] "Error in h(simpleError(msg, call)) : \n  error in evaluating the argument 'x' in selecting a method for function 't': requires nu
meric/complex matrix/vector arguments\n"
        <simpleError in h(simpleError(msg, cal
Calls: runApp ... Reduce -> lapply -> FUN -> %>% -> Reduce -> .safelapply
Execution halted

**Comment by rcorces** [comment]
ShinyArchRUiO is a separate package not managed by us
